src/styles/baseThemes/darkBaseTheme.js

```diff
@@ -19,5 +19,6 @@ export default {
     disabledColor: ColorManipulator.fade(Colors.fullWhite, 0.3),
     pickerHeaderColor: ColorManipulator.fade(Colors.fullWhite, 0.12),
     clockCircleColor: ColorManipulator.fade(Colors.fullWhite, 0.12),
+    shadowColor: Colors.fullWhite,
```

Comment:
I believe paper elements with white shadow show up better on dark backgrounds. If you don't like it I can put the default back to fullBlack. I would just to keep shadowColor in the palette so that it can be overridden. So, shall I set it to black again ?